_targets.R

```diff
@@ -93,10 +107,11 @@ p2_targets_list <- list(
 
 ## Visualize
 p3_targets_list <- list(
+  ## changed to object target
   tar_target(
-    figure_1_png,
+    figure_1,
     plot_nwis_timeseries(fileout = "3_visualize/out/figure_1.png", site_data_styled),
-    format = "file"
```

Comment:
Rm the format=file and also the png from the target name. Still, I get the same output, with the right figure in the `3_visualize/out/` folder.

Reply:
Nice! I love that this gives you the ggplot object as a target. That means you could add to it or modify it in later targets. I could see this being super helpful. For instance, if you wanted to make a bunch of different versions of the image to test out different styles, font sizes, etc, you could have all of them depend on this target.

The only sticking point here is that while this is now returning an object, and you have changed the name to reflect that, it is still writing a PNG file. Can you please separate this functionality into a file target, which takes the object and a filename as its inputs?